src/Facility.LanguageServer/Facility.LanguageServer.csproj

```diff
@@ -1,14 +1,14 @@
-<Project Sdk="Microsoft.NET.Sdk">
-
-  <PropertyGroup>
-    <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
-    <LangVersion>Latest</LangVersion>
-  </PropertyGroup>
-
-  <ItemGroup>
-    <PackageReference Include="Facility.Definition" Version="1.2.0" />
-    <PackageReference Include="OmniSharp.Extensions.LanguageServer" Version="0.7.7" />
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="2.0.0" />
-  </ItemGroup>
-</Project>
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Newline change? I've taken to using `* text=auto eol=lf` in `.gitattributes` (see other Facility projects).

Reply:
I think I copied `.gitattributes` and `.editorconfig` from FacilityCSharp, which doesn't have the line ending settings.